modules/rules_sh/0.3.0/MODULE.bazel

```diff
@@ -0,0 +1,15 @@
+module(
+    name = "rules_sh",
+    version = "0.3.0",
+    compatibility_level = 0,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.0.3")
+bazel_dep(name = "platforms", version = "0.0.4")
+
+sh_configure = use_extension("//bzlmod:extensions.bzl", "sh_configure")
+
+use_repo(sh_configure, "local_posix_config")
+use_repo(sh_configure, "rules_sh_shim_exe")
```

Comment:
nit, no need to change this: `use_repo` takes arbitrarily many positional arguments, so you can merge these calls.